index.bs

```diff
@@ -32,6 +32,33 @@ spec: ABNF; urlPrefix: https://tools.ietf.org/html/rfc5234
     text: VCHAR; url: appendix-B.1
     text: WSP; url: appendix-B.1
 
+spec: CSP; urlPrefix: https://www.w3.org/TR/CSP3/
+  type: dfn
+    text: strip url for use in reports; url: strip-url-for-use-in-reports
+
+spec: DOM; urlPrefix: https://dom.spec.whatwg.org/
+  type: dfn
+    text: document url; url: concept-document-url
+
+spec: INFRA; urlPrefix: https://infra.spec.whatwg.org
+  type: dfn
+    text: list; url: list
+
+spec: HTML; urlPrefix: https://html.spec.whatwg.org/C
+  type: dfn
+    text: policy container; url: policy-container
+    text: global object; url: global-object
+    text: window; url: window
+    text: WorkerGlobalScope; url: workerglobalscope
+
+spec: FETCH; urlPrefix: https://fetch.spec.whatwg.org/
+  type: dfn
+    text: request; url: concept-request 
+    text: response; url: concept-response 
+    text: header list; url: concept-response-header-list
+    text: header; url: concept-header
+    text: main fetch; url: main-fetch
```

Comment:
Almost none of this should be needed. Most of these terms are already exported. You just need to use the appropriate `for` attribute value and such.